os/gfile/gfile_z_example_cache_test.go

```diff
@@ -0,0 +1,41 @@
+// Copyright GoFrame Author(https://goframe.org). All Rights Reserved.
+//
+// This Source Code Form is subject to the terms of the MIT License.
+// If a copy of the MIT was not distributed with this file,
+// You can obtain one at https://github.com/gogf/gf.
+
+package gfile_test
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/gogf/gf/v2/os/gfile"
+)
+
+func ExampleGetContentsWithCache() {
+	// init
+	var (
+		fileName = "gflie_example.txt"
+		tempDir  = gfile.TempDir("gfile_example_cache")
+		tempFile = gfile.Join(tempDir, fileName)
+	)
+
+	// write contents
+	gfile.PutContents(tempFile, "goframe example content")
+
+	// read contents
```

Comment:
It reads the file content with cache duration of on minute,
which means it reads from cache after then without any IO operations within on minute.